api/cpp/esp-idf/slint/README.md

```diff
@@ -1,7 +1,57 @@
 # Slint
 
-A Declarative GUI toolkit
+[![Component Registry](https://components.espressif.com/components/slint/slint/badge.svg)](https://components.espressif.com/components/slint/slint)
```

Comment:
This is a link to itself.  As the README will be shown in this page.
But it does makes sense for the README seen on github.

Reply:
It’s inspired by this: https://github.com/espressif/esp-bsp/blob/f85c66f20486ccedfa7a26126b42d2774fa488ab/components/esp_lvgl_port/README.md?plain=1#L3